doc/source/ray-observability/user-guides/debug-apps/general-troubleshoot.rst

```diff
@@ -4,12 +4,12 @@ General Troubleshooting
 =======================
 
 Ray sometimes has some aspects of its behavior that might catch
-users off guard. There may be sound arguments for these design choices.
+users off guard (there may be sound arguments for these design choices). For example, users think of Ray as running on their local machine, and
+while this is sometimes true, this can lead to various issues.
 
-In particular, users think of Ray as running on their local machine, and
-while this is mostly true, this doesn't work.
+Here we list some common issues people may run into.
 
-Environment variables are not passed from the driver to workers
+Environment variables are not passed from the driver process to worker processes
```

Comment:
Can you actually promote runtime env env vars here to support this?

Reply:
In the fix below, it recommends runtime env. What else do you need?

Reply:
I will add an example here

Reply:
actually the example below seems good enough